tests/Application/src/Tests/Controller/SubscriptionUiTest.php

```diff
@@ -177,6 +177,23 @@ public function it_allows_accepting_a_subscription(): void
         $this->assertSame('accepted', $subscription->getState());
     }
 
+    /** @test */
+    public function it_allows_accepting_multiple_subscription(): void
+    {
+        $this->loadFixturesFromFile('subscriptions.yml');
+
+        $this->client->request('GET', '/admin/subscriptions');
+        $this->client->submitForm('Bulk accept');
+
+        $this->assertResponseRedirects(null, expectedCode: Response::HTTP_FOUND);
+
+        /** @var Subscription $subscription */
+        $subscription = static::getContainer()->get('app.repository.subscription')->findOneBy(['email' => 'marty.mcfly@bttf.com']);
+
+        $this->assertNotNull($subscription);
+        $this->assertSame('accepted', $subscription->getState());
```

Comment:
As this is "bulk" update we should probable check in the test state of multiple subscriptions to be sure it works correctly

Reply:
Also, can we somehow test the scenario where only some of existing subscriptions are bulk accepted (e.g. have 3 subscriptions in fixtures and accept only 2 of them)?

Reply:
done for the two of them.

Reply:
ah god, I failed with your second one. I've added one of them that is already accepted. That's not simple cause it was a lot easier to bulk all the items. Could you convert this to an issue on the board? We already know that it works cause I use this dev branch for my slides tests. But we need to refactor these tests and templates for that.